index.html

```diff
@@ -11851,11 +11851,11 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="state" id="aria-hidden">
 			<sdef>aria-hidden</sdef>
 			<div class="state-description">
-				<p><a>Indicates</a> whether the <a>element</a> is exposed to an accessibility API. See related <sref>aria-disabled</sref>.</p>
+				<p><a>Indicates</a>, when set to true, that an <a>element</a> and its entire subtree are hidden from assistive technology, regardless of whether it's visibly rendered. See related <sref>aria-disabled</sref>.</p>
```

Comment:
while the comparision to aria-disabled was pre-existing, I tend to agree with Dominic's comment for its removal.
```suggestion
				<p><a>Indicates</a>, when set to <code>true</code>, that an <a>element</a> and its entire subtree are hidden from assistive technology, regardless of whether it is visibly rendered.</p>
```